recipes/dsistudio/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "dsistudio" %}
+{% set version = "2021.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/frankyeh/DSI-Studio/archive/refs/tags/2021.10.05.zip
+    sha256: 6a9d1d2e249063a3bafc1fa6a99eebe776b35bcb1fd3034c24d290240be6ecfa
+    folder: src
+  - url: https://github.com/frankyeh/DSI-Studio-atlas/archive/refs/tags/2021.10.zip
+    sha256: 0ea27148512608674fe57e6eb10e523787202fb1f4de30d61d6d973e7c7f0367
+    folder: atlas
+  - url: https://github.com/frankyeh/TIPL/archive/refs/tags/2021.10.04.zip
+    sha256: b5926185484543c2067cdd1d20787e0372fd01636632fd5a87a5f358225bf765
+    folder: src/tipl
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - jom                            # [win]
+    - {{ cdt('mesa-libgl-devel') }}  # [unix]
+    - {{ cdt('mesa-dri-drivers') }}  # [unix]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxcb') }}            # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxfixes') }}         # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('libxext') }}           # [linux]
+    - {{ cdt('libxau') }}            # [linux]
+    - {{ cdt('expat-devel') }}       # [linux]
+    - zlib                           # [linux]
+    - libglu                         # [linux]
+    - boost-cpp
+  
+  host:
+    - {{ cdt('mesa-libgl-devel') }}  # [unix]
+    - qt 5.12.*                                   
+    - libglu                         # [linux]
+    - gcc 9.*                        # [linux]
+    - gxx 9.*                        # [linux]
+    - boost-cpp
+    - zlib
+  
+  run:
+    - qt 5.12.*
+    - libglu                         # [linux]
+    - ucrt                           # [win]      
+test:
+  commands:
+    - copy C:\bld\win-64\dsistudio*.tar.bz2 C:\Miniconda\conda-bld\win-64\           # [win]
+app:
+  entry: ${PREFIX}/dsi_studio.exe                 # [win]
+  entry: ${PREFIX}/dsi_studio                     # [linux]
+  entry: open ${PREFIX}/dsi_studio.app            # [osx]
+  summary: Diffusion MRI Tractography Tool
+  type: desk
+
+about:
+  home: http://dsi-studio.labsolver.org
+  license: GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE
```

Comment:
```
about:
  license_file:
    - LICENSE
    - vendor-licenses/
```
Could you please add a license to the atlas repository and then include all three license files (from each repo) in the about section? Then remove the LICENSE file you have added here in the recipe. We prefer to pull the LICENSE from the source zips because then if they change, they are automatically updated with each new build.

Reply:
Thanks for the suggestion! The atlas repo includes mostly 3rd party atlases all from different sources, and each of them has its own license/agreement in the same folder saved as *.note.txt. It seems I could not put a root license for the atlas repo.

For the tool, I will put the license at the right location.

Reply:
It's not strictly necessary to collect all the licenses into one file, but you do need to list each file in the `about/license_file` section so that they are all included with the end package.